File: src/main/java/com/haozileung/web/service/logging/ILoginLogService.java
package com.haozileung.web.service.logging;

import com.haozileung.infra.pager.PageResult;
import com.haozileung.web.domain.logging.LoginLog;

public interface ILoginLogService {

    void save(LoginLog dictionary);

    void update(LoginLog dictionary);

    void delete(LoginLog dictionary);

    PageResult<LoginLog> query(LoginLog dictionary, int pageNo, int pageSize);
}

File: src/main/kotlin/com/haozileung/web/service/logging/ILoginLogService.kt
package com.haozileung.web.service.logging

import com.haozileung.infra.page.PageResult
import com.haozileung.web.domain.logging.LoginLog

interface ILoginLogService {

    fun save(dictionary: LoginLog)

    fun update(dictionary: LoginLog)

    fun delete(dictionary: LoginLog)

    fun query(dictionary: LoginLog, pageNo: Int, pageSize: Int): PageResult<LoginLog>
}
